frontend/src/views/Market.vue

```diff
@@ -441,6 +445,17 @@ export default Vue.extend({
       this.nftPricesById = {};
     },
 
+    onSortChange( e: any) {
+      if (e === 'Ascending Price') {
```

Comment:
this can be cleaned up by doing `const sort = e === 'Ascending Price' ? 'Ascending' : 'Descending'` then reusing that variable.